bin/docserv-createconfig

```diff
@@ -306,30 +308,32 @@ def parse_cli(cliargs=None):
     parser = argparse.ArgumentParser(description=__doc__)
     parser.add_argument("-l", "--languages",
                         dest="langs",
-                        help=("Uses one or more languages in the format "
-                              "'language_country'. "
-                              "Upper and lower case is irrelevant."
+                        help=("One or more translation languages in the format "
+                              "'language-country'. "
+                              "Uppercase and lowercase are irrelevant."
                               ),
                         )
-    parser.add_argument("-d", "--default",
+    parser.add_argument("-d", "--default", "--default-language",
                         dest="default_lang",
-                        help="Default language",
+                        help=("Default language in the format "
+                              "'language-country'. "
```

Comment:
```suggestion
                              "'language-country' or 'language_country'. "
```

Reply:
True, we accept this format but we need correct it later. Imo this can stay a secret, much like the fact that we accept space-separated values now. :)